lib/arke/open_orders.rb

```diff
@@ -0,0 +1,53 @@
+require 'rbtree'
+require 'order'
+
+module Arke
+  class OpenOrders
+    def initialize
+      @book = {
+        buy: ::RBTree.new,
+        sell: ::RBTree.new
+      }
+    end
+
+    def contains?(side, price)
+      !@book[side][price].nil?
+    end
+
+    def add_order(order, id)
+      @book[order.side][order.price] = { id: id, order: order }
```

Comment:
why not doing id => order to facilitate search

Reply:
in this structure we will lookup by price more often, id so for ability to update and delete created orders, but it is not used for search, and price is